Channels/Operation/02_ImportOffers.md

```diff
@@ -0,0 +1,250 @@
+
+        > [Info] A single box is displayed for each attribute set available in *Omni-Channel*.
+
+4. Click the [CONTINUE] button in the bottom right corner.   
+    The *Adjust mappings* wizard window is displayed.
+
+    ![Adjust mappings](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/AdjustMappings.png "[Adjust mappings]")
+
+5. Configure the following settings:
+
+    + Click the *Channels set* drop-down list and select the *Omni-Channel* attribute set for mapping. All available *Omni-Channel* attribute sets are displayed in the list.       
+
+    + Click the *PIM attribute set* drop-down list and select the *PIM* attribute set for mapping. All available *PIM* attribute sets are displayed in the list.     
+    The *Mapping from "Omni-Channel attribute set name" to "PIM attribute set name"* section is displayed below the *Select channels set and PIM attribute set* box.
+
+        ![Adjust mappings mapping](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/AdjustMappingsMapping.png "[Adjust mappings mapping]")
+
+    + Adjust the attribute mapping from the selected *Omni-Channel* attribute set to the selected *PIM* attribute set as desired.   
+
+        > [Info] A default mapping is preconfigured by all drivers supporting the offer import.
+
+6. Click the [CONTINUE] button in the bottom right corner.   
+    The *Select matching attributes* wizard window is displayed.
+
+    ![Select matching attributes](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/SelectMatchingAttributes.png "[Select matching attributes]")
+
+7. Select the checkboxes of all PIM attributes that should be considered for matching with an existing PIM product. All mapped attributes are displayed in the box.
+
+    > [Info] A single box with PIM attributes is displayed for each attribute set mapping.
+
+8. Click the [CONTINUE] button in the bottom right corner.   
+    The *Matching and behavior after import* wizard window is displayed.
+
+    ![Matching and behavior after import](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/MatchingBehaviorAfterImport.png "[Matching and behavior after import]")
+
+    The selected match attributes have been saved and the *Match attributes saved* pop-up window is displayed.
+
+    ![Match attributes saved](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/MatchAttributesSaved.png "[Match attributes saved]") 
+
+9. Configure the following settings:
+
+    + Click the *Mode* drop-down list in the *Behaviour if a matched PIM product has been found* box and select the appropriate behaviour. The following options are available:
+        - **Link only**   
+        - **Link and transfer data**       
+
+    + Click the *Mode* drop-down list in the *Behaviour on change of the PIM product* box and select the appropriate behaviour. The following options are available:
```

Comment:
maybe be more clear that this configures the ETL change mode from the "link" (=dependency) that will be created between the pim products and offers (regardless of existing pim products were linked to offers or if new pim products were created for an offer because no matching pim product could be found)